addon/src/-internal/build-registry.ts

```diff
@@ -100,9 +100,6 @@ export default function buildRegistry(resolver: Resolver) {
 
   // @ts-ignore: this is private API.
   const fallbackRegistry = Application.buildRegistry(namespace);
-  // TODO: only do this on Ember < 3.13
-  // @ts-ignore: this is private API.
-  fallbackRegistry.register('component-lookup:main', Ember.ComponentLookup);
```

Comment:
This hasn't done anything for a long time now, and in any case this would be re-registering the same thing that was already there.